scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+while test $# -gt 0; do
```

Comment:
[nit] I don't think you need a `while` loop without doing `shift`. You could replace `while` with

```sh
if [[ "$#" -gt 0 ]]; then
```

or maybe even simpler remove `while` and modify `"$1"` expansion to use the default empty string (untested, but I hope it works)

```sh
case "${1:-}" in
```

Reply:
The expansion works, very cool. I learned something new about bash again.